packages/kbn-securitysolution-list-api/src/list_api/index.ts

```diff
@@ -57,6 +57,8 @@ export type {
   ImportListParams,
 } from './types';
 
+const version = '2023-10-31';
+
 const findLists = async ({
```

Comment:
looks like one of the APIs still misses version header

```
http://localhost:5601/kbn/api/lists/_find?page=1&per_page=5&sort_field=created_at&sort_order=desc
```

```
{
    "statusCode": 400,
    "error": "Bad Request",
    "message": "Please specify a version via elastic-api-version header. Available versions: [2023-10-31]"
}
```

Reply:
Added here: https://github.com/elastic/kibana/pull/165658/commits/6544f979be3b7789c097cd61747074d4e9f02f49
Thanks for checking that!